repos/testUtils/support/getClientWorld.js

```diff
@@ -0,0 +1,23 @@
+const { tryRequireSync } = require('@keg-hub/jsutils/src/node')
+const path = require('path')
+
+/**
+ * Searches the client's support directory for a world export
+ * @return {Object?} - the client's world object, or undefined if it does not exist
+ */
+const getClientWorld = () => {
```

Comment:
This is a super clean implementation. Not an easy thing to do, but you solved it well